### PR TITLE
some booty improves

### PR DESCRIPTION
**Pull request by austinzuffi**
checkkit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+## link
+live @ [softboys.club](https://softboys.club)
```
Comment (joelkperkins) [inline README.md:2]: oh nice good idea

File: src/App.js [hunk 2]
```diff
@@ -17,10 +17,9 @@ const Body = styled.div`
   align-items: center;
 `;
 
-const siteUrl = 'https://icecast.softboys.club:18000';
-const ssl = (listenurl) => {
-  let trackpath = !listenurl ? null : listenurl.slice(listenurl.lastIndexOf('/'));
-  return siteUrl + trackpath;
+const sslUrl = (trackUrl) => {
```
Comment (joelkperkins) [inline src/App.js:20]: - blocker
need to update the function calls after changing the function name

on line  36/37 ssl is undefined.

File: src/Components/Radio/Radio.component.jsx [hunk 3]
```diff
@@ -164,14 +164,15 @@ const Radio = ({activeTrack, setActiveTrack}) => {
   const statusBank = {
     EMPTY: 'Please Insert a Cassette!',
     PLAYING: `Now Playing: ${nowPlaying}`,
-    LOADED: `Press Play?`
+    LOADED: 'Press Play?',
+    LOADING: 'Loading...1 sec'
```
Comment (joelkperkins) [inline src/Components/Radio/Radio.component.jsx:168]: The transition from LOADING to LOADED breaks the audio stream on both tapes for me

**Comment by austinzuffi** [comment]
Okie im on the road. Had trouble fighting with git and obviously i didnt
test after solving all the merges. Lesson learned

**Comment by austinzuffi** [comment]
hiii! checkkkit, removed a line that was causing glitch playback. and included the variable that escaped me in merge conflicts. all goods now

**Comment by joelkperkins** [comment]
`const sslUrl = (trackUrl) => {
  let keyUrl = !trackUrl ? null : trackUrl.slice(trackUrl.lastIndexOf('/'));
  return process.env.REACT_APP_ICECAST_URL + keyUrl;
};`

the key url is undefined when i start up the branch

**Comment by austinzuffi** [comment]
checking out now

**Comment by austinzuffi** [comment]
> `const sslUrl = (trackUrl) => { let keyUrl = !trackUrl ? null : trackUrl.slice(trackUrl.lastIndexOf('/')); return process.env.REACT_APP_ICECAST_URL + keyUrl; };`
> 
> the key url is undefined when i start up the branch

is this something you're seeing in the react browser tools? or do you want me to declare the `let` outside of the scope of sslUrl definition

**Comment by joelkperkins** [comment]
`xhr.js:178 GET http://localhost:3000/undefined/status-json.xsl 404 (Not Found)`

not a matter of standard, it is undefined ^ so there is nowhere to go

**Comment by austinzuffi** [comment]
> `xhr.js:178 GET http://localhost:3000/undefined/status-json.xsl 404 (Not Found)`
> 
> not a matter of standard, it is undefined ^ so there is nowhere to go



> `xhr.js:178 GET http://localhost:3000/undefined/status-json.xsl 404 (Not Found)`
> 
> not a matter of standard, it is undefined ^ so there is nowhere to go

i'm not seeing that. on slack if you get a chance

**Comment by joelkperkins** [comment]
MERGERERED

**Pull request merged**